ursh/indico_ursh/client/index.js

```diff
@@ -46,7 +63,8 @@ function _validateAndFormatURL(url) {
     }
 
     // regular expression, because I.E. does not like the URL class
-    const re = RegExp(`^(.+:)//([^/]+)(/.*)?$`);
+    // provides minimal validation, leaving the serious job to the server
+    const re = RegExp(`^([\\d\\w]+:)//([^/ .]+(?:\\.[^/ .]+)*)(/.*)?$`);
```

Comment:
no need for backticks, you don't interpolate anything. fwiw, i would use `/.../` instead of `new RegExp()` (are you even supposed to use the constructor without `new`?), even if that requires some escaping of forward slashes. that's mostly a matter of preference though